ngrinder-controller/src/main/resources/controller-properties.map

```diff
@@ -41,3 +41,6 @@ controller.inactive_client_time_out,30000
 controller.enable_statistics,false
 controller.csv_separator,comma,
 controller.github_base_url,https://api.github.com,
+controller.ignore_too_low_tps,false,
+controller.ignore_too_many_errors,false,
+controller.ignore_too_much_traffic_on_region,false,
```

Comment:
It's a bit difficult to notice that the `controller-properties.map` file needs to be updated as well. But you didn't miss it. 👍

Reply:
thanks :)